docs/bugs/bug-234-auth-nav-user-button-missing-clerk-provider.md

```
@@ -0,0 +1,43 @@
+# BUG-234: AuthNav Can Render Clerk UserButton Without an Active ClerkProvider
+
+## Status: Actionable
+
+## Severity: Low
+
+## Summary
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)